src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,278 @@
+
+::RM-giftshop-trade-nickel
+{ (if: $triedNecklace is 1)[(set: $auntieFocus to "shifted back to the TV")]
+  (else:)[(set: $auntieFocus to "remained on the TV")]
+  (if: $gameOfChance is "dice set")[(set: $tradedItemPhrase to "Them dice is")]
+  (else-if: $gameOfChance is "dominoes")[(set: $tradedItemPhrase to "Them dominoes is")]
+  (else:)[(set: $tradedItemPhrase to "That knife’s")]
+}Auntie’s full attention $auntieFocus and she waved me over with her hand, fingers wiggling.
+
+“Now that’s what Auntie’s talkin ‘bout! Bring that nickel on over here; sloppier the joe, the better. Go ahead ‘n take the $gameOfChance — an’ I don’t wanna see you tryin’ to bring that back!” Auntie pointed her other hand to a great big sign on the wall proclaiming <b>ALL SALES FINAL</b>. “$tradedItemPhrase yours now.”
+
+<i>Not for long</i>, I thought as I pressed the wooden nickel into Auntie’s open palm. <i>Next stop, Toe</i>.
+
+> [[Head back to The Gutter|The-Gutter-start]] (<i>note: not a real link atm</i>)
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+::RM-junkheap-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-junkheap-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
+  (if: $playerName's last is "s")[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNames to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|RM-junkheap-entrance-nameprompt]].]
+
+::RM-junkheap-entrance
+It wasn’t really a heap, but that’s what we all called it. The “heap” was a bunch of beat-up shopping carts and folding tables ringed by crooked shelves, all loaded with castoffs from the camp.
+
+It was easy enough to pick through: whenever the heap ran out of flat surfaces, new carts, shelves and tables showed up in no time. Plus everything was laid out pretty neat. (Some folks said Auntie tidied up the heap in her spare time, but I’d never seen her more than 20 feet from her easy-chair.)
+
+Everybody wanted or needed something from the heap sooner or later, and only real assholes violated that pact and made a mess of it.
+
+[[I quickly skimmed through the dregs—|RM-junkheap-stuff]]
+
+::RM-junkheap-stuff
+Dead car batteries—
+Rusty hatchets left outside the night before a surprise storm—
+Bent folding chairs sporting dried bloodstains—
+Old-school Psychopathic t-shirts and Hatchet Gear ninjas had gotten too fat or skinny for—
+Rows and rows of cracked commemorative bongs with airbrushed Wicked Clown artwork (“<i>Violent J, 1972-20XX, Rest In Paint</i>”).
```

Comment:
This list seems like a good spot to use the `click:` macro for a one-by-one reveal.

Reply:
Definitely agreed! Only held off on doing that on this pass for time reasons.